packages/cli/README.md

```diff
@@ -0,0 +1,3 @@
+# sketch-assistant-cli
+
+Sketch Assistants Node command line utility.
```

Comment:
In line with terminology on docs (and [Wikipedia](https://en.wikipedia.org/wiki/Command-line_interface)):

```suggestion
Sketch Assistants Node command-line utility.
```